### ISSUE TITLE
Confused about gaussian atomization

### ISSUE THREAD

**Issue by alvin528** [RongLiu-Leo/AtomGS#3]
Great work! In your paper, it is mentioned that once the Gaussian points are atomized, their scales are fixed and no longer optimized. I couldn't find the corresponding implementation in the code. Could you please let me know if there's any information I might be missing?

**Comment by RongLiu-Leo** [comment]
Please refer to line 111 in train.py and line 211 in gaussian_model.py.

**Comment by alvin528** [comment]
> Please refer to line 111 in train.py and line 211 in gaussian_model.py.

@RongLiu-Leo Sorry, my point is how to fix the scale after the Gaussian points are atomized. There is no parameter detach operation, so the scale will still be optimized in subsequent steps.

**Comment by RongLiu-Leo** [comment]
Yes, you are right. We don't explicitly detach the learnable parameters. We use with torch.no_grad() env to constrain the Gaussian scales in this non-differentiable way.
